slack-notifications_5bec40412c7d3a31944df735.md

```diff
@@ -217,6 +217,16 @@ organization by using the [sem command line tool](https://docs.semaphoreci.com/a
 See the [sem command line tool](https://docs.semaphoreci.com/article/53-sem-reference)
 for further details.
 
+## Setting up Slack notifications through the UI
```

Comment:
Since we are describing how user can manage (create, edit, delete) the notification through UI, I'd like to suggest renaming the title to reflect this.